Library/Homebrew/requirements/glibc_requirement.rb

```diff
@@ -13,7 +13,7 @@ def initialize
 
   def self.system_version
     return @system_version if @system_version
-    libc = ["/lib/x86_64-linux-gnu/libc.so.6", "/lib64/libc.so.6", "/lib/libc.so.6", "/lib/i386-linux-gnu/libc.so.6", "/lib/arm-linux-gnueabihf/libc.so.6"].find { |s|
+    libc = ["/lib/x86_64-linux-gnu/libc.so.6", "/lib64/libc.so.6", "/lib/libc.so.6", "/lib/i386-linux-gnu/libc.so.6", "/lib/arm-linux-gnueabihf/libc.so.6", "/usr/lib64/libc.so.6", "/usr/lib/libc.so.6"].find { |s|
```

Comment:
Move `/usr/lib64/libc.so.6` immediately after `/lib64/libc.so.6` and remove `/usr/lib/libc.so.6`